src/http-aws4.js

```diff
@@ -51,6 +51,11 @@ const argv = yargs
       default: null,
       defaultDescription: '<auto>'
     },
+    profile: {
+      description: 'AWS profile',
+      requiresArg: true,
+      default: 'default'
```

Comment:
Sorry about that. Fixed and tested on EC2 to make sure the behavior hasn't changed.